Readme and deploy_azdo_pipelines change (#914)
Azure-Samples/modern-data-warehouse-dataops: e2e_samples/parking_sensors/README.md

    - [Databricks](#databricks)
    - [Data Factory](#data-factory)
- [How to use the sample](#how-to-use-the-sample)
  - [Setup](#setup)
    - [Prerequisites](#prerequisites)

>     - [Deployment Environment](#deployment-environment)

Reviewer: can we just call this bullet `Deployment Options` instead of Deployment environment? it seems neater (there is some confusion with Deployment and Configuration too IMO)

      - [Software Prerequisites if you use Dev Container](#software-prerequisites-if-you-use-dev-container)

>       - [Software Prerequisites if you do not use dev container](#software-prerequisites-if-you-do-not-use-dev-container)

Reviewer: please capitalize Dev Container like the previous line.

>   - [Configuration and Deployment](#configuration-and-deployment)

Reviewer: can we just use `Deployment` here?

## How to use the sample

> This sample has certain prerequisites for deployment, which will be detailed below. Please note that using the development container is optional. However, we recommend using the development container because it streamlines the setup process, ensures consistency across environments, and simplifies dependency management.

Reviewer: please change `development container` to `Dev Container`.

I would simplify this paragraph and just say:
`This sample has certain prerequisites for deployment, which are detailed below. We recommend using a Dev Container to streamline the setup process, ensure consistency across environments, and simplify dependency management.`

> This sample has certain prerequisites for deployment, which will be detailed below. Please note that using the development container is optional. However, we recommend using the development container because it streamlines the setup process, ensures consistency across environments, and simplifies dependency management.

Reviewer: you haven't fixed this block, it is still small caps and you should use Dev Container instead as that is the feature name

> This sample has certain prerequisites for deployment, which will be detailed below. Please note that using the development container is optional. However, we recommend using the development container because it streamlines the setup process, ensures consistency across environments, and simplifies dependency management.

Author: @elenaterenzi strange... I was pretty sure I fixed... but I did not in fact, sorry for that.
now is like this...I tried to remove the redundancy for dev container that I had before too:

"This sample requires certain prerequisites for deployment, which will be outlined below. While using the dev container is optional, we highly recommend it as it simplifies the setup process, ensures consistency across environments, and makes managing dependencies easier."

> This sample has certain prerequisites for deployment, which will be detailed below. Please note that using the development container is optional. However, we recommend using the development container because it streamlines the setup process, ensures consistency across environments, and simplifies dependency management.

Author: changed also the Dev Container to have the capital ones

### Setup

Following the setup prerequisites, permissions, and deployment environment options.

#### Prerequisites

> 1. [Github account](https://github.com/) : If you do not have already, create your GitHub Account.

Reviewer: change `If you do not have already` with `If you do not have one already`

> 1. [Github account](https://github.com/) : If you do not have already, create your GitHub Account.

Reviewer: ```suggestion
1. [Github account](https://github.com/) : If you do not have one already, create a GitHub Account.
```

> 2. [Azure Account](https://azure.microsoft.com/en-us/free/)

Reviewer: For consistency, I would suggest adding the same sentence we have for GH: `If you do not have one already, create an Azure account`

> 2. [Azure Account](https://azure.microsoft.com/en-us/free/)

Reviewer: ```suggestion
2. [Azure Account](https://azure.microsoft.com/en-us/free/) If you do not have one already, create an Azure Account.
```
For consistency, I would suggest adding the same sentence we have for GH

   - *Permissions needed*: ability to create and deploy to an azure [resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview), a [service principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals), and grant the [collaborator role](https://docs.microsoft.com/en-us/azure/role-based-access-control/overview) to the service principal over the resource group.
3. [Azure DevOps Project](https://azure.microsoft.com/en-us/products/devops/) : Follow the documentation to create a new project, or use an existing project you wish to deploy these resources to.
   - *Permissions needed*: ability to create [service connections](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml), [pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started/pipelines-get-started?view=azure-devops&tabs=yaml) and [variable groups](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml).

> #### Deployment Environment

Reviewer: as I mentioned, I think just using `Deployment Options` for the title would be simpler

> #### Deployment Environment

Reviewer: ```suggestion
#### Deployment Options
```

> As mentioned earlier, there are two options for setting up the deployment environment. The prerequisites for each option will be outlined next. We recommend using the development container as it simplifies the setup process, ensures consistency, and reduces configuration overhead.

Reviewer: I would suggest simplifying the sentence:
`As mentioned earlier, there are two deployment options. We recommend using a Dev Container to streamline the setup process, ensure consistency across environments, and simplify dependency management.`

> As mentioned earlier, there are two options for setting up the deployment environment. The prerequisites for each option will be outlined next. We recommend using the development container as it simplifies the setup process, ensures consistency, and reduces configuration overhead.

Reviewer: ```suggestion
As mentioned earlier, there are two deployment options. 
We recommend using a Dev Container to streamline the setup process, ensure consistency across environments, and simplify dependency management.
```

- [jq](https://stedolan.github.io/jq/)
- [makepasswd](https://manpages.debian.org/stretch/makepasswd/makepasswd.1.en.html)

> ### Configuration and Deployment

Reviewer: I would suggest changing the title to just `Deployment`

> ### Configuration and Deployment

Reviewer: ```suggestion
### Deployment
```

> Proceed in configure the environment variables as outlined below, fork the GitHub repository, and log into your Azure account before proceeding with the deployment.

Reviewer: do we need this sentence?

Author: So I am just adding sentences to introduce topics. I believe it is more organize and clear, instead of having a topic and nothing said.

Reviewer: it seems a bit of a repeat. if you want to keep this, can you please rephrase as it's not grammatically correct right now (maybe instead of "Proceed in configure" you meant "proceed with configuring"?)

Author: yeahp.... I can do that... no worries...I will rephrase.
